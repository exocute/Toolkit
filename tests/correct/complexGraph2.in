Graph Sport.Lisboa.e.Benfica
	Import Sporting.Clube.de.Portugal
	Export dois.um
	Activity goloSLB1 Eduardo.Salvio:24:Rafael.Silva
		Import Sporting.Clube.de.Portugal
		Export protestos.jogadores.do.Sporting
	Activity goloSLB2 Raul.Jimenez:47:Nelson.Semedo
		Import protestos.jogadores.do.Sporting
		Export contra.ataque.mortifero
	Activity goloSCP1 Bas.Dost:69:Joel.Campbell
		Import contra.ataque.mortifero
		Export dois.um
	connection goloSLB1->goloSLB2
	connection goloSLB2->goloSCP1
	