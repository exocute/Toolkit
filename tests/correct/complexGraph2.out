Name: Sport.Lisboa.e.Benfica
ImportName: Sporting.Clube.de.Portugal
ExportName: dois.um
Id: goloSLB1, Name: Eduardo.Salvio, Import: Sporting.Clube.de.Portugal, Export: protestos.jogadores.do.Sporting, Parameters: 24,Rafael.Silva
Id: goloSLB2, Name: Raul.Jimenez, Import: protestos.jogadores.do.Sporting, Export: contra.ataque.mortifero, Parameters: 47,Nelson.Semedo
Id: goloSCP1, Name: Bas.Dost, Import: contra.ataque.mortifero, Export: dois.um, Parameters: 69,Joel.Campbell
goloSLB1:goloSLB2
goloSLB2:goloSCP1